packages/api/src/utils/zod.ts

```diff
@@ -44,12 +43,15 @@ export const numberFormatSchema = z.object({
 export const timeChartSeriesSchema = z.object({
   table: z.optional(sourceTableSchema),
   type: z.literal('time'),
-  aggFn: aggFnSchema,
+  aggFn: AggregateFunctionSchema,
+  level: z.number().optional(),
```

Comment:
Not sure if it matters but `level` represents a percentile value so it's more than just a number, it also needs to be between [0, 1]. Not sure if we want to try to bake that into the schema or not. I don't believe it is in common-utils either but doesn't seem like it holds up to user input.

Reply:
good callout ty

Reply:
Addressed